docs/storage/object/index.md

```diff
@@ -1,37 +1,30 @@
 
-# LUMI-O: The Object Storage
-
+# Object storage - LUMI-O
 
 !!! warning
-    This page is about a partition that is not yet available
-
-
+    This page is about a hardware partition that is not yet available.
 
-In an object-based storage, data are managed as objects instead of being 
-organized as files in a directory hierarchy. 
+The LUMI-O object store offers a total of 30 PB storage for storing, sharing,
+and staging of data.
 
-<!-- This partition may be used for storing, sharing and
-staging your data. It's based on Ceph and has a storage capacity of 30 PB. -->
+In an object-based storage, data are managed as objects instead of being
+organized as files in a directory hierarchy.
 
```

Comment:
The buckets are created and accessed with secure key pairs that are valid for a (user defined) time interval. (More information about this will be added before LUMI-O is accessible.)

It is recommended to rather create several buckets than to fill one bucket close to it's file quota limit.

Reply:
Alright, I think we can safely leave these detail for a later revision of the LUMI-O documentation.